app/models/sam_account_reckoner.rb

```diff
@@ -24,8 +26,12 @@ def should_clear?
     user.persisted? && user.duns_number_changed?
   end
 
-  def user_in_sam?
-    client.duns_is_in_sam?(duns: user.duns_number)
+  def sam_status
+    if client.duns_is_in_sam?(duns: user.duns_number)
```

Comment:
This is why I get a little bit leery of enums, especially if we are inspecting the DB at some point. In any event, can we map these to constants or something?